client-js/app/scripts/client_rpc.ts

```diff
@@ -0,0 +1,94 @@
+import promise = require('es6-promise');
+
+var Promise = promise.Promise;
+
+export function sendRequest(method: string, params: Object, socket, server: string): Promise<string> {
+        console.log("sending dumb RPC named: "+ method+" argument: "+ JSON.stringify(params));
+        return socket.call("org.labrad."+server+"." + method, params)
+                      .then((result) => <string>result);
+}
```

Comment:
I've added ```sendRequest``` to ```JsonRpcSocket```, I defined the prefix in app.ts when new JsonRpcSocket is defined. Let me know if that makes sense...